DEVELOPING_OPAL/demos/src/main/scala/org/opalj/ai/InfiniteRecursions.scala

```diff
@@ -139,27 +139,29 @@ object InfiniteRecursions extends ProjectAnalysisApplication {
                         // in the same manner; the idea is to reduce false positives
                         // due to non-infinite recursions due to side effects
                         if (callOperands.forall {
-                            case domain.DomainSingleOriginReferenceValueTag(v) =>
-                                if (v.origin < 0 /* === the value is a parameter*/ ||
-                                    // the value is always created anew (no sideeffect)
-                                    body.instructions(v.origin).opcode == NEW.opcode)
-                                    true
-                                else
-                                    false
-                            case v: domain.AnIntegerValue =>
-                                if (localsArray(0).exists(_ eq v))
-                                    true // the value is parameter
-                                else
-                                    false
-                            case v: domain.ALongValue =>
-                                if (localsArray(0).exists(_ eq v))
-                                    true // the value is parameter
-                                else
-                                    false
-                            case _: domain.LongSet      => true
-                            case _: domain.IntegerRange => true
-                            case _                      => false
-                        })
+                                case domain.DomainSingleOriginReferenceValueTag(v) =>
+                                    if (v.origin < 0 /* === the value is a parameter*/ ||
+                                        // the value is always created anew (no sideeffect)
+                                        body.instructions(v.origin).opcode == NEW.opcode
+                                    )
+                                        true
+                                    else
+                                        false
+                                case v: domain.AnIntegerValue =>
+                                    if (localsArray(0).exists(_ eq v))
+                                        true // the value is parameter
+                                    else
+                                        false
+                                case v: domain.ALongValue =>
+                                    if (localsArray(0).exists(_ eq v))
+                                        true // the value is parameter
+                                    else
+                                        false
+                                case _: domain.LongSet      => true
+                                case _: domain.IntegerRange => true
+                                case _                      => false
+                            }
+                        )
```

Comment:
In this case, the opening braces (on the if statement and forall) are on one line and the closing braces are on two different lines. Can this be made consistent?

Reply:
Closing braces/parens on one line I don't think are possible. Opening ones on two lines might be possible by manually formatting them that way, scalafmt might not revert that.